hackathon/models.py

```diff
@@ -112,15 +113,14 @@ class HackProject(models.Model):
     "scores" has been moved to HackProjectScore. See comments there."""
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
-    # Each model can only be created by one user: One To Many
-    created_by = models.ForeignKey(User,
-                                   on_delete=models.CASCADE,
-                                   related_name="hackprojects")
+    created_by = models.ForeignKey(User,on_delete=models.SET_NULL, null=True, blank=True, related_name="hackproject", default="")
```

Comment:
This looks more than 79 characters. Can you split the line up please?
Also, please remove the null=True as you are already defaulting it to an empty string which I would keep.